setup.py

```diff
@@ -18,8 +18,7 @@ def load_version():
     # load all vars into globals, otherwise
     #   the later function call using global vars doesn't work.
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict)
+    with open(os.path.join('nilearn', 'version.py')) as fp:        exec(fp.read(), globals_dict)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use of exec](https://www.codacy.com/app/lesteve/lesteve_nilearn/file/822040796/issues/source?bid=2752673&fileBranchId=2752902#l21)